src/drivers/uavcannode/UavcanNode.cpp

```diff
@@ -521,6 +522,20 @@ void UavcanNode::Run()
 		}
 	}
 
+	// safety -> standard::indication::button
+	if (_safety_sub.updated()) {
+		safety_s safety;
+
+		if (_safety_sub.copy(&safety)) {
+			if (safety.safety_switch_available) {
+				standard::indication::Button Button{};
+				Button.button = standard::indication::Button::BUTTON_SAFETY;
+				Button.press_time = safety.safety_off ? UINT8_MAX : 0;
```

Comment:
Should the value of `press_time` be tracked and incremented here?

Reply:
I was actually going to change that entirely now.